src/PyCall.jl

```diff
@@ -59,7 +59,13 @@ typealias PyPtr Ptr{PyObject_struct} # type for PythonObject* in ccall
 #########################################################################
 # Wrapper around Python's C PyObject* type, with hooks to Python reference
 # counting and conversion routines to/from C and Julia types.
+"""
+This is a reference to a Python Object. Julia objects may be converted to Python Objects using `PyObject(juliavar)` and vice-versa using `convert(T, o::PyObject)`.
```

Comment:
"Object" should not be capitalized.

They can be converted to native julia types `T` with `convert(T, o)` or `T(o)`.